src/types/state.js

```diff
@@ -51,7 +51,27 @@ export type RightClickedMarker = {|
   +markerIndex: MarkerIndex,
 |};
 
-export type ProfileViewState = {|
+/**
+ * Full profile view state
+ * They should not be used from the active tab view.
+ * NOTE: This state is empty for now, but will be used later, do not remove.
+ * globalTracks and localTracksByPid states will be here in the future.
+ */
+export type FullProfileViewState = {||};
```

Comment:
This is empty for now. We are gonna put the globalTracks and localTracksByPid here.